engine/include/cubos/engine/plugins/settings_inspector.hpp

```diff
@@ -0,0 +1,15 @@
+#ifndef CUBOS_ENGINE_PLUGINS_SETTINGS_INSPECTOR_HPP
+#define CUBOS_ENGINE_PLUGINS_SETTINGS_INSPECTOR_HPP
+
+#include <cubos/engine/cubos.hpp>
+
+using namespace cubos::engine;
+
+namespace cubos::engine::plugins
+{
+    /// Plugin that allows inspecting the current settings
+    /// @param cubos CUBOS. main class
+    void settingsInspectorPlugin(Cubos& cubos);
```

Comment:
I would just move this into another namespace and directory, maybe `plugins::tools`? Not sure what we would name it.